engine/src/test/java/io/zeebe/engine/processor/StreamProcessorTest.java

```diff
@@ -575,7 +572,7 @@ public void onRecovered(final ReadonlyProcessingContext context) {
   }
 
   @Test
-  public void shouldNotCreateSnapshotsIfNoProcessorProcessEvent() throws Exception {
+  public void shouldNotCreateSnapshotsIfNoProcessorProcessEvent() {
```

Comment:
When we could get rid of this limitation then we would also get updates from exporter position.

Reply:
What limitation are you referring to?

Reply:
That we do not take snapshots if the position has not changed. I mean we could also just do snapshot every 15min regardless of the position

Reply:
This is kind of expensive right now because we push snapshots, so with large states this is a lot of extra work for little gain.

That said, it's really tempting to remove it - then we dont have the issue with the exporters. Maybe we should discuss this a bit more thoroughly, it's a little out of scope here imho but a very good point

Reply:
Yes we can do it separately. Came just in my mind when I saw this code :sweat_smile:

Reply:
https://github.com/zeebe-io/zeebe/issues/4017